Source/WebInspectorUI/UserInterface/Views/FolderizedTreeElement.js

```diff
@@ -298,25 +282,16 @@ WI.FolderizedTreeElement = class FolderizedTreeElement extends WI.GeneralTreeEle
         if (settings.topLevel)
             return this;
 
-        var folder = this._folderTypeMap.get(settings.type);
+        let folder = this._folderTypeMap.get(settings.type);
         if (folder)
             return folder;
 
-        folder = createFolderTreeElement.call(this, settings);
+        let folderId = `${settings.type}-${settings._folderSettingsKey}`;
+        folder = new WI.FolderTreeElement(settings.displayName, settings.representedObject, {id: folderId});
```

Comment:
Style: instead of creating a `folderId` variable, we normally inline it
```js
folder = new WI.FolderTreeElement(settings.displayName, settings.representedObject, {
    id:`${settings.type}-${settings._folderSettingsKey}`,
});
```